src/kern/npf_connkey.c

```diff
@@ -99,8 +99,8 @@ connkey_setkey(npf_connkey_t *key, uint16_t proto, const void *ipv,
 	}
 }
 
-static inline void
-connkey_getkey(const npf_connkey_t *key, uint16_t *proto, npf_addr_t *ips,
+inline void
```

Comment:
In C, it generally makes no sense to define `inline`, but not `static` function. AFAIK, it is legal (after all, `inline` is merely a _hint_ and compiler has a full right to completely ignore it), but I think under no circumstances compiler would try to inline such function. So, in this case you can just drop the `inline`.